src/__tests__/resources/assets.integration.spec.ts

```diff
@@ -1,162 +1,179 @@
 // Copyright 2019 Cognite AS
 
-import CogniteClient from '../../cogniteClient';
-import { CogniteError } from '../../error';
-import { CogniteMultiError } from '../../multiError';
-import { Asset } from '../../types/types';
-import { randomInt, setupLoggedInClient } from '../testUtils';
-
-describe('Asset integration test', () => {
-  let client: CogniteClient;
-  beforeAll(async () => {
-    client = setupLoggedInClient();
-  });
-  const rootAsset = {
-    name: 'test-root',
-    description: 'Root asset for cognitesdk-js test',
-    metadata: {
-      refId: 'test-root',
-    },
-    externalId: 'test-root' + randomInt(),
-  };
-
-  const childAsset = {
-    name: 'test-child',
-    description: 'Child asset for cognitesdk-js test',
-    metadata: {
-      refId: 'test-child',
-    },
-    parentExternalId: rootAsset.externalId,
-  };
-  let assets: Asset[];
-
-  test('create', async () => {
-    assets = await client.assets.create([childAsset, rootAsset]);
-    expect(assets[0].createdTime).toBeInstanceOf(Date);
-    expect(assets[0].lastUpdatedTime).toBeInstanceOf(Date);
-  });
+import {
```

Comment:
This file should not be touched?

Reply:
My bad, I copied the wrong one from v1, but should be correct now 👌🏼